pennylane/transforms/zx/converter.py

```diff
@@ -336,7 +336,7 @@ def processing_fn(res):
             else:
                 expanded_operations.append(op)
 
-        expanded_tape = QuantumScript(expanded_operations, mapped_tape.measurements, [])
+        expanded_tape = QuantumScript(expanded_operations, mapped_tape.measurements)
```

Comment:
We probably need a test for this.

Reply:
We can edit `pytest.ini` to turn the deprecation warning into an error

Reply:
Yep. Let's place:
```
    error:The `prep` keyword argument:UserWarning:pennylane.tape.qscript
```
at the end of `tests/pytest.ini`.

Reply:
I have done what you say here. `pytest.ini` is a document I don't know, can you confirm that it is what you were asking for?